WordPress/Classes/ViewRelated/Blog/QuickStartTours.swift

```diff
@@ -31,9 +32,15 @@ extension QuickStartTour {
         }
     }
 
-    var shownInBlogDetails: Bool {
+    var taskCompleteDescription: NSAttributedString? {
         get {
-            return true
+            return nil
+        }
+    }
+
+    var possibleEntryPoints: Set<QuickStartTourEntryPoint> {
```

Comment:
[?] I can see that almost all tours have the possible entry points specified. Also from my understanding having no possible entries and having both entries, ultimately get treated the same way.

So I wonder if this is the best way to minimize human errors in the future when adding tours.

I would suggest to completely remove the default implementation. Or to keep it but return `[.blogDetails, .blogDashboard]`. I feel this is more verbose. Also we can add a comment that we only need to override this if the tour can only be accessed from a specific place.

What do you think?

Reply:
Good point. I decided to completely remove the default implementation to make this property more intentional d4b9b41